src/Components/benchmarkapps/Wasm.Performance/TestApp/Shared/ComplexTable/TableComponent.razor

```diff
@@ -26,21 +26,8 @@
     [Parameter]
     public List<string> Columns { get; set; }
 
-    DateTime t1;
-    DateTime t2;
-    Task RefreshComponent(int index)
+    Task HandleClickEvent(int index)
     {
-        t1 = DateTime.Now;
-        StateHasChanged();
         return Task.CompletedTask;
     }
-    protected override Task OnAfterRenderAsync(bool firstRender)
-    {
-        if (!firstRender)
-        {
-            t2 = DateTime.Now;
-            Console.WriteLine("Refresh Time " + (t2 - t1).TotalMilliseconds);
-        }
-        return base.OnAfterRenderAsync(firstRender);
-    }
```

Comment:
This event handler logic change has no impact on the benchmarks because they never called this code anyway. I'm just clearing it up a bit.